tests/query/parser/unit_tests/test_parse_snql_query_initial.py

```diff
@@ -0,0 +1,971 @@
+from typing import Type
+
+import pytest
+
+from snuba.datasets.entities.entity_key import EntityKey
+from snuba.datasets.entities.factory import get_entity
+from snuba.query import OrderBy, OrderByDirection, SelectedExpression
+from snuba.query.composite import CompositeQuery
+from snuba.query.data_source.simple import Entity
+from snuba.query.dsl import Functions as f
+from snuba.query.dsl import and_cond, column, in_cond, literal, or_cond
+from snuba.query.logical import Query
+from snuba.query.parser.exceptions import ParsingException
+from snuba.query.snql.parser import parse_snql_query_initial
+
+test_cases = [
+    pytest.param(
+        "MATCH (events) SELECT foo(1) AS `alias`, bar(2) AS `alias`",
+        Query(
+            from_clause=Entity(
+                EntityKey.EVENTS, get_entity(EntityKey.EVENTS).get_data_model()
+            ),
+            selected_columns=[
+                SelectedExpression("alias", f.foo(literal(1), alias="alias")),
+                SelectedExpression("alias", f.bar(literal(2), alias="alias")),
+            ],
+            array_join=None,
+            condition=None,
+            groupby=None,
+            having=None,
+            order_by=None,
+            limitby=None,
+            limit=1000,
+            offset=0,
+            totals=False,
+            granularity=None,
+        ),
+    ),
+    pytest.param(
+        "\n           MATCH (events)\n           SELECT test_func(release) AS test_func_alias,\n              event_id BY project_id, platform\n           WHERE timestamp >= toDateTime('2021-01-01T00:00:00') AND timestamp < toDateTime('2021-01-02T00:00:00') AND project_id = 1\n        ",
```

Comment:
Can you fix this up?

Reply:
do i have to 😢

Reply:
yes, there are not that many. A simple:

`s/\\n           / /g` should do

Reply:
ok, you know your regex, very useful skill